mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerConsistencyTest.java

```diff
@@ -417,7 +422,7 @@ void deleteMailboxByIdShouldBeConsistentWhenMailboxDaoFails(CassandraCluster cas
                 }));
             }
 
-            @Test
+            @RepeatedTest(100)
```

Comment:
isn't this a leftover of diagnosing the issue ?

Reply:
No that's a mistake